huxley/api/serializers/school.py

```diff
@@ -72,20 +72,45 @@ class Meta:
 
 
     def validate(self, data):
+        invalid_fields = {}
         international = data.get('international')
         primary_phone = data.get('primary_phone')
         secondary_phone = data.get('secondary_phone')
-
-        if primary_phone:
+        beginner_delegates = data.get('beginner_delegates')
+        intermediate_delegates = data.get('intermediate_delegates')
+        advanced_delegates = data.get('advanced_delegates')
+        spanish_speaking_delegates = data.get('spanish_speaking_delegates')
+        chinese_speaking_delegates = data.get('chinese_speaking_delegates')
+
+        total_delegates = sum((
+            beginner_delegates or 0,
+            intermediate_delegates or 0,
+            advanced_delegates or 0))
+
+        def validate_phone(phone, international):
             if international:
-                validators.phone_international(primary_phone)
+                validators.phone_international(phone)
             else:
-                validators.phone_domestic(primary_phone)
+                validators.phone_domestic(phone)
+
+        if primary_phone:
```

Comment:
The removal of this line runs afoul of several unit tests, and it preexisted before this PR (removed line 79), so I left it in in case it serves a purpose I am unaware of. However, it does seem unnecessary since primary_phone is a required field.